front/app/containers/Admin/pagesAndMenu/containers/ContentBuilder/components/Widgets/_shared/AdminPubsCarrousel/AdminPubCard/index.tsx

```diff
@@ -0,0 +1,154 @@
+import React from 'react';
```

Comment:
Same here - could we use the full name instead for this component? E.g. AdminPublicationCard